plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationSummaryDialog.java

```diff
@@ -46,15 +53,31 @@ public NewApplicationSummaryDialog(Shell parentShell, CreateApplicationFromTempl
 
 	@Override
 	protected void createAreaAfterResourceSummary(Composite parent) {
-		if(job.getParameters().size() == 0) {
-			return;
-		}
 		
 		Composite area = new Composite(parent, SWT.NONE);
 		GridDataFactory.fillDefaults()
 				.align(SWT.FILL, SWT.FILL).grab(true, true).applyTo(area);
 		GridLayoutFactory.fillDefaults().margins(10, 10).applyTo(area);
 		
+		final IBuildConfig buildConfig = findBuildConfigWithWebHooks();
+		if (buildConfig != null) {
+			Link webHooksLink = new Link(area, SWT.NONE);
+			webHooksLink.setText("See how you can add webhooks to your git repository <a>here</a>.");
```

Comment:
I'm not certain about the text.  It leads me to believe I'm going to be told how to actually add the hooks to my repo, but it looks like we are just exposing hooks that you use to do some other thing too.  I think, in the plugin as a whole, we need to find ways to add 'help' that links to the OpenShift docs where appropriate for things like this: "Take this url, go to github, click settings,...", etc.